commands/Moderation/warn.js

```diff
@@ -50,23 +50,25 @@ class Warn extends Command {
         data.guild.save();
 
         let caseInfo = {
-            channel: message.channel,
-            moderator: message.author,
+            channel: message.channel.name,
```

Comment:
It's better to save the ID of the channel (`message.channel.id`) instead of the name